e2e/deno-e2e/tests/deno-integrated.spec.ts

```diff
@@ -272,6 +273,117 @@ console.log('123');
     }, 120_000);
   });
 
+  describe('application --bundler deno_emit', () => {
+    it('should create deno app', async () => {
```

Comment:
This project doesn't actually use deno_emit as the bundler from what I can tell since the flag is never passed in. it's using esbuild in that case.

I'd like to have a e2e test for each bundler.
You only need to test the build for the different bundlers and not test/lint/serve.